client/app/components/project/package-section.js

```diff
@@ -21,6 +21,16 @@ export default class ProjectPackageSectionComponent extends Component {
     return payablePackage;
   }
 
+  get invoices() {
+    return this.args.packages
+      .mapBy('invoices')
+      .reduce((acc, curr) => acc.concat(curr));
+  }
+
+  get hasPayableInvoices() {
+    return this.invoices.filter((invoice) => invoice.statuscode === 'Approved').length >= 1;
```

Comment:
So an "Approved" invoice is one that has yet to be paid? Or one that has already been paid?

Reply:
Yep, that's it. It's confusing.